internal/pkg/fleetapi/action.go

```diff
@@ -135,6 +138,16 @@ func (a *ActionUnknown) OriginalType() string {
 	return a.originalType
 }
 
+func (a *ActionUnknown) AckEvent() AckEvent {
+	return AckEvent{
+		EventType: "ACTION_RESULT", // TODO Discuss EventType/SubType needed - by default only ACTION_RESULT was used - what is (or was) the intended purpose of these attributes? Are they documented? Can we change them to better support acking an error or a retry?
+		SubType:   "ACKNOWLEDGED",
```

Comment:
The initial implementation of `constructEvent` would always return this `EventType` and `SubType`. Were there plans to use these in another way? Should we consider it for acking errors or retries or have these attributes been silently deprecated?
cc @kpollich @cmacknz

Reply:
This discussion should take place in the openapi spec pr: https://github.com/elastic/fleet-server/pull/2180